pkg/proxy/proxy.go

```diff
@@ -199,10 +200,9 @@ func (p *Proxy) newReverseProxy(ctx *gin.Context, target *namespace.NamespaceAcc
 		}
 	}
 	return &httputil.ReverseProxy{
-		Director:       director,
-		Transport:      transport,
-		FlushInterval:  -1,
-		ModifyResponse: addCorsToResponse,
```

Comment:
I'm a bit concerned with modifying the response headers before we redirect the request to the api server instead of using a response modifier. In this case our CORS headers can be overridden by the api server based on the default strict api server settings.

Reply:
Good point - I didn't realize that. :man_facepalming: